views/scss/comments-panel.scss

```
@@ -100,6 +111,173 @@
+    &:hover,
+    &:focus {
+        background: #e7ebef;
+    }
+
+    &.item-comments-rich-tool--active {
+        background: #d9e8f4;
+        color: #1f4f73;
+    }
+
+    .item-comments-toolbar-icon {
+        font-family: 'tao';
```

Stylelint (17.14.0)

[error] Expected no quotes around "tao" (font-family-name-quotes)

(font-family-name-quotes)